app/scripts/controllers/transactions.js

```diff
@@ -215,6 +215,7 @@ module.exports = class TransactionController extends EventEmitter {
     }
     txParams.gasPrice = ethUtil.addHexPrefix(gasPrice.toString(16))
     txParams.value = txParams.value || '0x0'
+    if (txParams.to === null) delete txParams.to
```

Comment:
may need to handle the case where `txParams.to === '0x'` if we're not handling that somewhere else. if we are handling that somewhere else, we should put all this params normilization in the same place.

Reply:
( mew was sending us `to: 0x` once upon a time )

Reply:
eyk okay

Reply:
shouldnt be an issue in this case we already remove to fields that would have that before this point